10up/classifai
# Add: WP Admin menu icon

### Description of the Change
Add admin menu icon for ClassifAI menu.


### Alternate Designs
Use gray icon instead of red.


### Benefits
ClassifAI menu is easier to reach and distinguish.


### Possible Drawbacks
n/a


### Verification Process

1. Login to admin.
2. See new icon next to ClassifAI menu.


### Checklist:






### Applicable Issues
Close #130
